src/v/cluster/node/local_monitor.cc

```diff
@@ -101,7 +101,7 @@ ss::future<struct statvfs> local_monitor::get_statvfs(const ss::sstring path) {
     }
 }
 
-float percent_free(const storage::disk& disk) {
+float local_monitor::percent_free(const storage::disk& disk) {
     long double free = disk.free, total = disk.total;
     return float((free / total) * 100.0);
 }
```

Comment:
nit: this doesn't seem to use any instance state. you could make it a static method